tests/FINDOLOGIC/Export/Tests/DataHelperTest.php

```diff
@@ -159,8 +186,55 @@ public function testAttributeKeyCharacterLimitCausesException()
      * @param int $stringLength The string length to generate.
      * @return string The multi byte character string.
      */
-    public function generateMultiByteCharacterString($stringLength)
+    public static function generateMultiByteCharacterString($stringLength)
     {
         return implode('', array_fill(0, $stringLength, '©'));
     }
+
+    /**
+     * @noinspection PhpMethodMayBeStaticInspection
+     *
+     * @return array
+     */
+    public function allValuesProvider()
```

Comment:
I forgot the methods in this file. Can you also add the return types for the remaining tests and dataproviders?